cmd/controller/cmd/root.go

```diff
@@ -97,11 +97,12 @@ func init() {
 	rootCmd.AddCommand(viper.GetConfigCommand())
 }
 
-func initConfig(_ *cobra.Command, _ []string) error {
+func initConfig(cmd *cobra.Command, _ []string) error {
 	configAccessor = viper.NewAccessor(config.Options{
-		StrictMode:  true,
+		StrictMode:  false,
```

Comment:
I don't think this is the complete fix here... I understand it'll allow it to pass... I think the more fundamental fix is to make InitializePFlags keep track of all the flags (and persistent flags) that it sees at the time of initialization. Once it does, it let those flags be skipped while everything else must be recognized (if strictMode is true)...

Reply:
@EngHabu is the reason for the problem here that, there are other modules registering flags, and or config, will not recognize them?